src/opm/output/eclipse/Summary.cpp

```diff
@@ -1042,7 +1043,6 @@ Summary::Summary( const EclipseState& st,
     // for simulation restart.
     {
         auto& rvec     = this->handlers->rstvec_backing_store;
-        auto& handlers = this->handlers->handlers;
 
```

Comment:
No.

Give this another name, but don't remove it.

Reply:
Why? It is referenced only once below. I find the use of the same name, nested, a bit weird, but if I see it in the code I won't be too confused when debugging...

Reply:
> Why? It is referenced only once below.

Avoiding double indirection at every reference.  Also, the line is shorter.

> Would summary_handlers be a better name?

I'd personally prefer `hndlrs` or something similarly succinct.

By the way, you should talk to @joakim-hove about removing the `input` unit convention.  This system was explicitly introduced in commit 2d8457e3 (PR #356) as a means of [enabling programmatic manipulation of simulation decks from Python](https://github.com/OPM/opm-common/pull/356#issuecomment-373325922).

Reply:
> By the way, you should talk to @joakim-hove about removing the input unit convention. This system was explicitly introduced in commit 2d8457e (PR #356) as a means of enabling programmatic manipulation of simulation decks from Python.

I am not removing it, just removing the remains of unused implementation details: the input unit convention uses straight 1.0 constants instead of reading them from this array for example, and I think that is fine.

Reply:
> I am not removing [the `input` unit convention], just removing the remains of unused implementation details

Unused only because they're not currently hooked up.  These arrays are what powers `UnitSystem::to_si()` and `UnitSystem::from_si()`